packages/http-specs/spec-summary.md

```diff
@@ -1742,42 +1749,6 @@ Expected response body:
 }
 ```
 
-### Payload_Pageable_ServerDrivenPagination_ContinuationToken_requestQueryResponseBody
```

Comment:
Why this part is deleted? Previously not synced?

Reply:
Nice catch. Let me make an investigation why this part is removed even if the test case is not deleted.

Reply:
Updated.